hazelcast/src/test/java/com/hazelcast/map/impl/MapSplitBrainTest.java

```diff
@@ -54,8 +53,8 @@
  * keys 1000..1999 -> value 2 (merged, result of (3-1))
  * keys 2000..2999 -> value 3 (not merged)
  */
-@RunWith(HazelcastParallelClassRunner.class)
-@Category({QuickTest.class, ParallelTest.class})
```

Comment:
No need to remove `ParallelTest` category, it's only used when running tests parallel in multiple JVMs.